File: Backend/src/main/java/pl/bucior/carrental/model/dto/UserDto.java
package pl.bucior.carrental.model.dto;

import lombok.Data;
import pl.bucior.carrental.model.enums.Role;

import java.io.Serializable;

@Data
public class UserDto implements Serializable {
    private Long id;
    private String email;
    private Role role;
    private String firstName;
    private String lastName;
    private String pesel;
    private String idCardNumber;
    private Boolean isActive;
}

File: Wpf/Gracjan/CarRentalWPF/CarRentalWPF.Library2/FromServerDto/UserDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CarRentalWPF.Library.FromServerDto
{
    public class UserDto
    {
        public int id;

        public string email;

        public string role;

        public string firstName;

        public string lastName;

        public string pesel;

        public string idCardNumber;

        public bool isActive;

        public int? agencyId;
    }
}
